mlrun/utils/notifications/notification/webhook.py

```diff
@@ -54,8 +56,12 @@ async def push(
         if override_body:
             request_body = override_body
 
+        ssl_context = ssl.SSLContext()
+        # Disable SSL verification if verify_ssl is False
+        ssl_context.verify_mode = ssl.CERT_NONE if not verify_ssl else ssl.CERT_REQUIRED
```

Comment:
I think this is relevant only if endpoint scheme is https
For http endpoint, is cert required work even if not served?